src/shared/utils/kms/index.ts

```diff
@@ -0,0 +1,29 @@
+import { Blob } from "node:buffer";
+import { TextEncoder } from "node:util";
+import { kms } from "./client";
+
+export const decryptKms = async (
+  encryptedBytes: Uint8Array,
+  context: {}
```

Comment:
Be cautious with using `{}` as a type, it's quite _flexible_

<img width="304" alt="Screenshot 2023-06-12 at 11 27 20" src="https://github.com/alphagov/di-billing-transaction-monitoring/assets/35689158/1740d63c-b998-4b01-9146-a81d64f20d60">